apis/stardust/types/index.d.ts

```diff
@@ -584,7 +584,7 @@ declare namespace stardust {
         /**
          * Function used by the visualization to tell Nebula to it wants to relinquish focus
          */
-        blur?(): void;
+        blur?(b?: boolean): void;

```

Comment:
you should not change direct in this file, but instead change the jsdoc source that is ised to generate it (and the json spec)